<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NutrientInfoStatusWidget</class>
 <widget class="QWidget" name="NutrientInfoStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Nutritive information status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLed" name="m_statusLed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="shape">
      <enum>KLed::Circular</enum>
     </property>
     <property name="look">
      <enum>KLed::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_statusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">&lt;status&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="m_updateButton">
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset theme="view-refresh">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="m_detailsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Details...</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-information"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
